File: GMgen.js
var adj  = new Array('multiversal', 'singularity', 'chronoplasmic', 'hexagonal', 'unimorphic', 'sound', 'numerical');
var desc = new Array('god', 'anti-matter', 'null', 'rainbow', 'dream', 'note', 'significance');
var noun = new Array('life', 'equation', 'pill', 'particle', 'totem', 'bullet', 'cycle');

function make_story(){
    var final = '';
    
    $([adj, desc, noun]).each(function(){
        var final += this[Math.floor(Math.random() * this.length)];
    });
}

File: GMgen.pl
#!/usr/bin/perl

use strict;
use warnings;

my @adj  = qw/multiversal singularity chronoplasmic hexagonal unimorphic sound numerical/;
my @desc = qw/god anti-matter null rainbow dream note significance/;
my @noun = qw/life equation pill particle totem bullet cycle/;

my $plot = '';

foreach  ( (\@adj, \@desc, \@noun) ){
    $plot .= @$_[rand(@$_)] . ' ';
}

print "$plot\n";